### PR TITLE
Templating: Speeds up certain variable queries for Postgres MySql MSSql

### LINKED ISSUE #28600: Dashboard variable fills slow if more than 250k unique values on PostgreSQL (TimescaleDB)
**What happened**: on PostgreSQL (TimescaleDB) dashboard variable fills slow if more than 250k unique values, on InfluxDB there is no problem with same 250k unique values, variable fills without a problem

**What you expected to happen**: dashboard variable fills fast on PostgreSQL (TimescaleDB) even if more than 250k unique values

**How to reproduce it (as minimally and precisely as possible)**: have a PostgreSQL (TimescaleDB) table with column of 250k unique values and try to fill dashboard variable with "select column from table"

**Anything else we need to know?**:

**Environment**:
- Grafana version: 7.3.0-beta2
- Data source type & version: PostgreSQL (TimescaleDB)
- OS Grafana is installed on: Debian 9
- User OS & Browser: Windows 10, Chrome 86.0.4240.111
- Grafana plugins: /
- Others: /

### PR DESCRIPTION
**Pull request by hugohaggmark**
**What this PR does / why we need it**:
As described in #28600

**Which issue(s) this PR fixes**:
Fixes #28600

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by hugohaggmark** [comment]
Hold merge until after the 7.3.1 release.

File: public/app/plugins/datasource/mssql/response_parser.ts [hunk 1]
```diff
@@ -69,14 +69,13 @@ export default class ResponseParser {
...
-        const value = rows[i][j];
-        if (res.indexOf(value) === -1) {
-          res.push(value);
-        }
+        res.push(rows[i][j]);
       }
     }
 
-    return _.map(res, value => {
+    const unique = Array.from(new Set(res));
```
Comment (dprokop) [inline public/app/plugins/datasource/mssql/response_parser.ts:76]: Clever:)

**Review by dprokop — APPROVED** [review]
🔥

File: public/app/plugins/datasource/mssql/response_parser.ts (hunk 1 above)
Comment (hugohaggmark) [reply public/app/plugins/datasource/mssql/response_parser.ts:76]: Thank you 🙏

**Comment by marefr** [comment]
@hugohaggmark have you tested all these data sources with devenv/dashboards? Otherwise I can help out with that.

**Comment by hugohaggmark** [comment]
> @hugohaggmark have you tested all these data sources with devenv/dashboards? Otherwise I can help out with that.

Thank you @marefr I'd really appreciate that because I have only tested Postgres and the specific query in the issue.

**Review by marefr — APPROVED** [review]
Did some quick manual testing of all three sql data sources using devenv and seems to work as expected.

LGTM

**Pull request merged**